client/components/survey-container/index.tsx

```diff
@@ -10,8 +11,12 @@ type SurveyContainerProps = {
 	onSkip: () => void;
 	onChange: ( questionKey: string, value: string[] ) => void;
 	hideBackOnFirstPage?: boolean;
+	hideContinue?: boolean;
+	hideSkip?: boolean;
```

Comment:
I think hideContinue and hideSkip are not needed here.

Reply:
Unless you want this to be the default at the survey level, in which case I think you should take it into consideration in:
```
hideContinue={ currentQuestionConfiguration?.hideContinue }
hideSkip={ currentQuestionConfiguration?.hideSkip }
```

Reply:
Indeed, they are unused, I'll remove them. Nice catch!